its/ruling/src/test/resources/expected/kotlin/kotlin-language-server/kotlin-S1874.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I created [SONARKT-251](https://sonarsource.atlassian.net/browse/SONARKT-251) to fix FP